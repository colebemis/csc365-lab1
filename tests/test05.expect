Enter query: Querying the average GPA for grade 1
2.995
Enter query: 