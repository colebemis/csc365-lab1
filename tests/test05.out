Enter query: TOWLEY, LANE, 54
Enter query: 